python-frontend/src/main/java/org/sonar/python/metrics/FileLinesVisitor.java

```diff
@@ -101,15 +101,21 @@ private void visitNode(SubscriptionContext ctx) {
     }
   }
 
-  private void handleDocString(@Nullable StringLiteral docstring) {
+  protected void handleDocString(@Nullable StringLiteral docstring) {
+    linesOfDocstring.addAll(countDocstringLines(docstring));
+  }
+
+  public static Set<Integer> countDocstringLines(@Nullable StringLiteral docstring) {
```

Comment:
The name of this method seems misleading to me. I would remove "count" from it.
Maybe something like `docStringLineNumbers` would be clearer.